webextension/scripts/background.js

```diff
@@ -514,16 +514,27 @@ chrome.tabs.onUpdated.addListener((tabId, info, tab) => {
   if (info.status === 'complete') {
     updateWaybackCountBadge(tab.id, tab.url)
     chrome.storage.local.get(['auto_archive', 'fact_check'], (event) => {
+      // fact check
+      if (event.fact_check === true) {
+        factCheckPage(tab.id, tab.url)
+      }
       // auto save page
       if (event.auto_archive === true) {
         auto_save(tab.id, tab.url)
       }
+    })
+  } else if (info.status === 'loading') {
+    updateWaybackCountBadge(tab.id, tab.url)
+    chrome.storage.local.get(['auto_archive', 'fact_check'], (event) => {
       // fact check
       if (event.fact_check === true) {
```

Comment:
The same code is repeated again in the "complete" state above ^! Please remove that.

Reply:
Okay